seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1180,10 +1180,12 @@
         "var myStr = \"FirstLine\\n\\t\\\\SecondLine\\nThirdLine\";"
       ],
       "tests": [
-        "assert(myStr === \"FirstLine\\n\\t\\\\SecondLine\\nThirdLine\", 'message: <code>myStr</code> should have encoded text with the proper escape sequences and no spacing.');",
-        "assert(myStr.match(/\\n/g).length == 2, 'message: <code>myStr</code> should have two newline characters <code>\\n</code>');",
-        "assert(myStr.match(/\\t/g).length == 1, 'message: <code>myStr</code> should have one tab character <code>\\t</code>');",
-        "assert(myStr.match(/\\\\/g).length == 1, 'message: <code>myStr</code> should have one correctly escaped backslash character <code>\\\\</code>');"
+        "assert(myStr.match(/ /g) === null, 'message: <code>myStr</code> should not contain any spaces');",
```

Comment:
Thanks for all of the feedback @Greenheart, really appreciate it! Making these changes now.

When I ran npm test it said 'assert.isFalse' isn't a function (...might be my syntax). I'm too tired to find out why tonight but will have a look tomorrow. If you don't think it matters too much I'll leave myStr.match in for now.